pkgs/os-specific/linux/zfs/generic.nix

```diff
@@ -184,6 +184,7 @@ let
 
       # Add Bash completions.
       install -v -m444 -D -t $out/share/bash-completion/completions contrib/bash_completion.d/zfs
+    '' + optionalString (lib.versionOlder version "2.2.6") ''
```

Comment:
This has broken the 2.1 build, btw.

`/nix/store/ffr5qkpiyl0bdz15w1b8hmqnclw6938h-stdenv-linux/setup: line 229: cd: /nix/store/hn1s5py5wfinj0k1wq7ghhzwv19ahc72-zfs-kernel-2.1.15-6.6.49/share/bash-completion/completions: No such file or directory`

Reply:
Yea, this is not conditional on `buildUser`. In fact, I actually think the line above it isn't even necessary in 2.2.*. Will add this to my zfs fixes pr: https://github.com/NixOS/nixpkgs/pull/340165